hazelcast/src/main/java/com/hazelcast/internal/networking/nio/NioChannelWriter.java

```diff
@@ -222,6 +248,11 @@ private void schedule() {
      * This call is only made by the IO thread.
      */
     private void unschedule() throws IOException {
+        if (forceUnschedule) {
+            unregisterOp(OP_WRITE);
+            scheduled.compareAndSet(true, false);
```

Comment:
You can call set since it is scheduled. You don't need to do a cas.

Reply:
Sure, I'll change it. Thanks.